### ISSUE TITLE
Verbose messages on connection close

### ISSUE THREAD

**Issue by bmario** [metricq/metricq-python#137]
When `close()` is complete, these messages appear. I'm not sure where they are coming from. It might even come from aio-pika.

```
Robust channel <RobustChannel "None#Not initialized channel"> has been closed.
NoneType: None
Robust channel <RobustChannel "None#Not initialized channel"> has been closed.
NoneType: None
```

**Comment by bmario** [comment]
Maybe this can be solved with #124

**Comment by tilsche** [comment]
This is really annoying.
